docs/contents/usage/README.md

```diff
@@ -217,17 +217,18 @@ You can configure `burrito` with environment variables.
 
 #### Controllers' configuration
 
-|            Environment variable             |                              Description                               |             Default              |
-| :-----------------------------------------: | :--------------------------------------------------------------------: | :------------------------------: |
-|         `BURRITO_CONTROLLER_TYPES`          |                      list of controllers to start                      |        `layer,repository`        |
-| `BURRITO_CONTROLLER_TIMERS_DRIFTDETECTION`  |              period between two plans for drift detection              |              `20m`               |
-|     `BURRITO_CONTROLLER_TIMERS_ONERROR`     |        period between two runners launch when an error occurred        |               `1m`               |
-|   `BURRITO_CONTROLLER_TIMERS_WAITACTION`    |        period between two runners launch when a layer is locked        |               `1m`               |
-| `BURRITO_CONTROLLER_LEADERELECTION_ENABLED` |               whether leader election is enabled or not                |              `true`              |
-|   `BURRITO_CONTROLLER_LEADERELECTION_ID`    |                   lease id used for leader election                    | `6d185457.terraform.padok.cloud` |
-| `BURRITO_CONTROLLER_HEALTHPROBEBINDADDRESS` |  address to bind the health probe server embedded in the controllers   |             `:8081`              |
-|   `BURRITO_CONTROLLER_METRICSBINDADDRESS`   |     address to bind the metrics server embedded in the controllers     |             `:8080`              |
-| `BURRITO_CONTROLLER_KUBERNETESWEBHOOKPORT`  | port used by the validating webhook server embedded in the controllers |              `9443`              |
+|            Environment variable                      |                              Description                                   |             Default              |
+| :--------------------------------------------------: | :------------------------------------------------------------------------: | :------------------------------: |
+|         `BURRITO_CONTROLLER_TYPES`                   |                      list of controllers to start                          |        `layer,repository`        |
+| `BURRITO_CONTROLLER_TIMERS_DRIFTDETECTION`           |              period between two plans for drift detection                  |              `20m`               |
+|     `BURRITO_CONTROLLER_TIMERS_ONERROR`              |        period between two runners launch when an error occurred            |               `1m`               |
```

Comment:
we should specify that we are talking about a controller error now. It could also be understood as a runner error. Same in `cmd/controllers/start.go`